mypy.ini

```diff
@@ -9,3 +9,4 @@ ignore_errors = false
 warn_redundant_casts = true
 allow_untyped_calls = true
 allow_any_generics = true
+ignore_missing_imports= true
```

Comment:
Why is this needed ? Could we filter out false positives instead if they are not too common ?

Reply:
You can override this specifically for the auth0 package then

Reply:
done